docker/config/magpie/magpie.ini

```diff
@@ -0,0 +1,166 @@
+###
```

Comment:
Maybe one way to avoid this whole file could be to override the docker command with a call to `sed` before starting the app to replace the `magpie.url` part ?

Reply:
The sed is really more messy, the CMD is using dockerfile env that would need to be hard-coded in the compose and if the Magpie command is updated it will break the compose.

I understand why you switch the precedence of the ini over the environment variable, but when Magpie is using the default ini, it should allow overriding it else the environment variables are of no use.